docs/buildah-run.md

```diff
@@ -26,7 +26,8 @@ The *path* to an alternate OCI-compatible runtime.
 
 Adds global flags for the container runtime. To list the supported flags, please
 consult manpages of your selected container runtime (`runc` is the default
-runtime, the manpage to consult is `runc(8)`)
+runtime, the manpage to consult is `runc(8)`).
+If the global flag takes in a string, it will be `--runtime-flag global-flag=value`.
```

Comment:
Add something like.  Make sure you do not pass the `--`.  If you want to pass --debug to runc, then the command to buildah would be `--runtime-flag debug`

Reply:
fixed.

Reply:
But if the prepending if not provided is a pain in the tookus, then +1 on Dan's thought.  Perhaps:

Do not include the leading dashes (--) for the runc arguments.  For instance to set the log format to json, the global flag would be "log-format=json".